packages/vscode-extension/src/utils/autoOpenHelper.ts

```diff
@@ -165,18 +165,22 @@ export async function ShowScaffoldingWarningSummary(
             apiSpecFilePathRes.error
           );
         } else {
-          message = generateScaffoldingSummary(
+          message = await generateScaffoldingSummary(
             createWarnings,
             teamsManifest,
-            path.relative(workspacePath, apiSpecFilePathRes.value[0])
+            path.relative(workspacePath, apiSpecFilePathRes.value[0]),
+            path.join(AppPackageFolderName, teamsManifest.copilotExtensions!.plugins![0].file),
+            workspacePath
           );
         }
       }
-      if (commonProperties.isApiME) {
-        message = generateScaffoldingSummary(
+      if (commonProperties.isApiME && teamsManifest.composeExtensions![0].apiSpecificationFile) {
```

Comment:
Using `teamsManifest.composeExtensions?.[0].apiSpecificationFile` may be better?

Reply:
it has checked commonProperties.isApiMe which have already made sure `teamsManifest.composeExtensions[0]` exists.